.env.docker

```
@@ -59,6 +59,10 @@ OTEL_SERVICE_NAME=coop
 NODE_ENV=production
 ITEM_QUEUE_TRAFFIC_PERCENTAGE='0'
 UI_URL=http://localhost:3000
+MRT_RECOVERY_LOOKBACK_DAYS=30
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MRT_RECOVERY_LOOKBACK_DAYS key should go before the NODE_ENV key

(UnorderedKey)